js/main.js

```diff
@@ -1,22 +1,128 @@
+  'Предлагаются в аренду шикарные, светлые апартаменты в башне Neva Towers. Полностью укомплектован дорогой мебелью и техникой.',
+  'Панорамное остекление по всему периметру апартаментов. Видовые характеристики: футуристический вид на город. В ночное время огни небоскребов не оставят Вас равнодушными.',
+  'Вместительная гардеробная комната и дополнительные системы хранения, одна мастер-ванная, один гостевой санузел с постирочной. Панорамное остекление во всех помещениях. Вид во внутренний двор.',
+];
+
+const PHOTOS = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+
+//Функция, возвращающая случайное целое.
 function getRandomNumber(min, max) {
-  if (min < max) {
-    min = Math.ceil(min);
-    max = Math.floor(max);
-    return Math.floor(Math.random() * (max - min + 1)) + min; //Максимум и минимум включаются
-  }
-  return 'Минимум должен быть меньше либо неравен Максимуму';
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
 }
-console.log(getRandomNumber(0,100));
-
-// Функция, возвращающая случайное число с плавающей точкой из переданного диапазона включительно.
-function getRandomNumber2(min, max, maxDigits) {
-  if (min < max) {
-    min = Math.ceil(min);
-    max = Math.floor(max);
-    const digitsDegree = 10 ** maxDigits;
-    return ((Math.random() * (max - min) + min) * digitsDegree) / digitsDegree;
+
+function getRandomNumber2(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  const numberWithZero = Math.floor(Math.random() * (max - min + 1)) + min;
+  if (numberWithZero > 9) {
+    return numberWithZero;
+  } else {
+    return (`0${numberWithZero}`);
   }
-  return 'Минимум должен быть меньше либо неравен Максимуму';
 }
-console.log(getRandomNumber2(0,100,5));
+
+
+// Функция, возвращающая случайное число с плавающей точкой.
+function getRandomLatLng(min, max, maxDigits) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  const digitsDegree = 10 ** maxDigits;
+  return ((Math.random() * (max - min) + min) * digitsDegree) / digitsDegree;
+}
+
+const getRandomArrayFeauters = (features) => {
+  const lastIndex = features.length - 1;
+  const a = getRandomNumber(0, lastIndex);
+  const b = getRandomNumber(0, lastIndex);
+  const min = Math.min(a, b);
+  const max = Math.max(a, b);
+
+  return features.slice(min, max);
+};
+
+const locationLat = getRandomLatLng(35.65000, 35.70000, 5);
+const locationLng = getRandomLatLng(139.70000, 139.80000, 5);
+
+const getRandomArrayElement = (elements) => elements[getRandomNumber(0, elements.length - 1)];
+
+
+const createPost = () => ({
+  author: {
+    avatar: `img/avatars/user${  getRandomNumber2(1, 10)  }.png`,
```

Comment:
в константы вынеси аргументы у функции
```suggestion
    avatar: `img/avatars/user${getRandomNumber2(1, 10)}.png`,
```
и лишних пробелов наставил. Также плохо называть функции getRandomNumber2 с числами на конце, да и не только функции, вообще плохо такой нейминг использовать